test/Testing.js

```diff
@@ -0,0 +1,153 @@
+
+// var db = require('./db');
+// var heroesClass = require('./heroes');
+// var assert = require('assert');
+
+const chai = require('chai'),
+	chaiHttp = require('chai-http'),
+	mocha = require('mocha'),
+	sinon = require('sinon'),
+	db = require('../db'),
+	heroesClass = require('../heroes'),
+	server = require('../server'),
+	assert = chai.assert;
+
+chai.should();
+chai.use(chaiHttp);
+
+describe('Heroes', function()
+{
+	describe('#getHeroes()', function()
+	{
+		it('should return all heroes', function()
+		{
+			const heroes = new heroesClass.Heroes();
+			assert.equal(db.heroes,heroes.getHeroes());
+		});
+	});
+
+	describe('#getHeroById()', function()
+	{
+		it('should return hero with same id', function()
+		{
+			const heroes = new heroesClass.Heroes();
+			assert.equal(db.heroes.find(element => element.id === 5),heroes.getHeroById(5));
```

Comment:
if some time we change our db and there no hero with id 5 - test will break
is that mean that heroes.find break?
use mock for data